test/e2e/testing-manifests/statefulset/nginx/statefulset.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1beta1
```

Comment:
apps/v1beta2?

Reply:
@foxish test will be owned by the sig-storage

Reply:
`statefulset_utils.go` is  using `AppsV1beta1` interfaces. We can update the all statefulset yamls along with upgrading version in `statefulset_utils.go` in separate PR.

Reply:
FWIW the client version used in statefulset_utils.go is irrelevant to the version used in yaml, because we have conversions between different versions. You can specify one version in yaml and get the resource back in another version. 

Note that `.spec.selector` is no longer defaulted from labels in apps/v1beta2, after switching, it needs to be specified explicitly in yaml.